src/components/OSCALCatalogGroup.js

```diff
@@ -24,6 +25,11 @@ const StyledListItemPaper = styled(Paper)`
   border-radius: 0.5em;
 `;
 
+const StyledListItemText = styled(ListItemText)(({ theme }) => ({
```

Comment:
```suggestion
const WithdrawnListItemText = styled(ListItemText)(({ theme }) => ({
```

Let's specify _how_ it's styled and what it actually represents. Imagine some day we want to signify "important" controls (or some other made up descriptor) somehow -- those would be styled too but perhaps in a different way. So `StyledListItemText` doesn't really get the point across in the same way.

I know we've made this mistake elsewhere, but let's do better here.